lib/commands/mongo.js

```diff
@@ -132,6 +139,307 @@ mongo.createUser = function (databaseName, username, password, isReadOnly, cb) {
 }
 
 // ------------------------------------------------------------------------------
+mongo.migrate = function (databaseName, version, cb) {
+  mongoController.databasesForUser(
+    userConfig.data.userId,
+    function (err, databases) {
+      var dbExists = false
+
+      if (err) {
+        err = error.handleApiError(err, 'GET_DATABASES', cb)
+        if (err.length > 0) {
+          return cb(err)
+        }
+      }
+      if (databases.length === 0) {
+        xervo.io.error('You currently have no databases. One can be created using the `mongo create` command.')
+        return cb()
+      }
+
+      if (databaseName !== '') {
+        for (var i = 0; i < databases.length; i++) {
+          if (databases[i].name === databaseName) {
+            dbExists = true
+          }
+        }
+
+        if (!dbExists) {
+          xervo.io.error('No database found by that name in this account. Please try again with the correct name or omit the name to choose from a list.')
+          return cb()
+        }
+      }
+
+      mongo.chooseDatabasePrompt(databases, databaseName, function (err, database) {
+        if (err) {
+          err = error.handleApiError(err, 'MIGRATE_MONGO', cb)
+          if (err.length > 0) {
+            return cb(err)
+          }
+        }
+        if (!database) {
+          return cb('You must select a database.')
+        }
+        xervo.io.print('Selected MongoDB database ' + database.name.data + '.')
+        xervo.io.print('Migrating MongoDB database.')
+
+        mongo.chooseRegionPrompt(function (err, regionData) {
+          if (err) {
+            err = error.handleApiError(err, 'MIGRATE_MONGO', cb)
```

Comment:
if `mongo.chooseRegionPrompt` handles the error, you can just pass it to the callback instead of doing another error handler